### ISSUE TITLE
[GDB][VSCode] Fails to update scope when changing frame 

### ISSUE THREAD

**Issue by maxmarsc** [maxmarsc/dave#8]
When selecting a new frame using the CLion UI while debugging with GDB, the call to [check_frame](https://github.com/maxmarsc/dave/blob/main/dave/server/debuggers/gdb/commands.py#L54) never gets the updated frame. It always stay stucked on the breakpoint's frame.

It seems to be an issue with CLion itself, a ticket has been created : https://youtrack.jetbrains.com/issue/CPP-41827/GDB-API-gdb.postevent-calls-to-frame-does-not-reflect-current-state

**Comment by maxmarsc** [comment]
Due to CLion's way of interactive with GDB/LLDB this might not be solved soon. There's currently no way to fetch CLion's selected frame from within the debugger API.

This might requires developing a custom CLion plugin to inform DAVE of any frame change

**Comment by sudara** [comment]
Am I right in assuming this means one cannot step through the debugger and see the changes, right now one would need to keep typing `dave show myVariable`?

**Comment by maxmarsc** [comment]
No, it means if you go up and down the stack using Clion UI, dave won't know about it

**Comment by sudara** [comment]
Ok, I think that's what I meant — right now when I hit "step over" in lldb then I currently lose the graph in Dave.
